cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -82,6 +82,7 @@ func NewCMServer() *CMServer {
 		KubeletConfig: client.KubeletConfig{
 			Port:        ports.KubeletPort,
 			EnableHttps: false,
+			HTTPTimeout: time.Duration(5) * time.Second,
```

Comment:
@ddysher @bgrant0607 

Thanks, I will refactor the PR to have this change alone. I still have a few unanswered questions though before I do that.